### ISSUE TITLE
Update build instructions

### ISSUE THREAD

**Issue by jorabin** [derecalliance/cryptography#9]
Picking up from #2.

Assuming Java code is removed (see #8), and put elsewhere, require clear build instructions for the remaining Rust library.

Needs usage examples.

**Comment by ehanoc** [comment]
@rsinha we moved the protobufs to another repo. Is this still relevant?

**Comment by jorabin** [comment]
> @rsinha we moved the protobufs to another repo. Is this still relevant?

which repo did you move to?

**Comment by rsinha** [comment]
@jorabin The protobufs are in https://github.com/derecalliance/protobufs.
The issue needs to be updated as protobufs are no longer duplicated -- the other asks are still relevant.

**Comment by jorabin** [comment]
> @jorabin The protobufs are in https://github.com/derecalliance/protobufs. The issue needs to be updated as protobufs are no longer duplicated -- the other asks are still relevant.

yes I know :) I thought @ehanoc was saying they'd been moved somewhere else.

**Comment by larkiny** [comment]
Split into two tickets; tidying project structure is now in [this issue](https://github.com/derecalliance/cryptography/issues/37)

**Comment by larkiny** [comment]
Will archive current cryptography repi
